trunner/harness/plo.py

```diff
@@ -124,14 +128,39 @@ def erase(self, device: str, offset: int, size: int, timeout: Optional[int] = No
             self.send_cmd("YES!")
             self.dut.expect_exact("Erased", timeout=timeout)
         except pexpect.TIMEOUT as e:
-            raise PloError("Wrong erase command output!", cmd=cmd, output=self.dut.before) from e
+            raise PloError(
+                "Wrong erase command output!", cmd=cmd, output=self.dut.before
+            ) from e
 
         try:
             self._assert_prompt()
         except PloError as e:
             e.cmd = cmd
             raise e
 
+    def cleanmarkers(self, device: str, clean: bool, cleanmarkers_args: tuple, timeout):
+        """Performs set jffs2 cleanmarkers command."""
+
+        if clean is True:
+            x = "-e"
+        else:
+            x = ""
+
+        cmd = (
+            f"jffs2 -d {device} {x} "
+            f"-c {cleanmarkers_args[0]}:{cleanmarkers_args[1]}:{cleanmarkers_args[2]}:{cleanmarkers_args[3]}"
+        )
```

Comment:
We rather use this approach for long string in python: https://github.com/phoenix-rtos/phoenix-rtos-tests/blob/8c6d062c2fbd327e2664ec371fcf8bddf1135483/trunner/harness/unity.py#L70C18-L70C18

Reply:
If you want try different approach, this could work:
```suggestion
cmd = f"jffs2 -d {device} {x} -c {':'.join(map(str, cleanmarkers_args))}"
```
This version can be more efficient because it avoids manual indexing: